binr/rax2/rax2.c

```diff
@@ -107,7 +107,7 @@ static int help() {
 		"  -D      base64 decode        ;\n"
 		"  -E      base64 encode        ;\n"
 		"  -f      floating point       ;  rax2 -f 6.3+2.1\n"
-		"  -F      stdin slurp code hex ;  rax2 -F < shellcode.c\n"
+		"  -F      stdin slurp code hex ;  rax2 -F < shellcode.[c(py]\n"
```

Comment:
lgtm

Reply:
Darn, the typo. meant. to write [c/py] instead of [c(py]. Sorry